test/index.test.js

```diff
@@ -24,54 +25,38 @@ describe('index', function() {
   describe('onPreBuild()', function() {
     beforeEach(function() {
       this.createCronAllowedRange = this.sandbox.stub();
-      const getConfigValue = this.sandbox.stub();
 
       const { onPreBuild } = proxyquire('../src', {
-        './lib/cron-allowed-range': this.createCronAllowedRange,
-        './lib/get-config-value': getConfigValue
+        './lib/cron-allowed-range': this.createCronAllowedRange
       });
 
       this.config = {
         expression: '* * * * *',
         timezone: 'America/Toronto'
       };
 
+      this.sandbox.stub(process.env, 'CONTEXT')
+        .value('production');
+
       this.sandbox.stub(process.env, 'DEPLOYMENT_HOURS_EXPRESSION')
         .value(this.config.expression);
 
       this.sandbox.stub(process.env, 'DEPLOYMENT_HOURS_TIMEZONE')
         .value(this.config.timezone);
 
-      const inputs = {
+      this.inputs = {
+        contexts: 'production',
         expression: '* * * * 1',
         timezone: 'Africa/Abidjan'
       };
 
-      getConfigValue
-        .withArgs({
-          configName: 'expression',
-          defaultValue: '* * * * *',
-          env: this.config.expression,
-          input: inputs.expression
-        })
-        .returns(this.config.expression);
-
-      getConfigValue
-        .withArgs({
-          configName: 'timezone',
-          defaultValue: 'America/Toronto',
-          env: this.config.timezone,
-          input: inputs.timezone
-        })
-        .returns(this.config.timezone);
```

Comment:
Calling the actual `get-config-value` function was fine, since we already set it up to return expected values.